README_Dockerfile.md

```diff
@@ -0,0 +1,80 @@
+# Dockerfile writing best practices
+
+* Use an appropriate / minimal base image that's actively maintained and responsive to security updates. `ubuntu:18.04` is fairly common, though [`minideb`](https://github.com/bitnami/minideb) is under review as a base image as it's minimal and well-maintained from a security standpoint.
+* Minimize number of stages being used - each directive like `ENV`, `ARG` can be separately cached
```

Comment:
Why is this important?

Reply:
This minimizes container layers ... and by increasing cache hit frequency, you reduce build times.

Reply:
Repeat build speed up because of caching?

Reply:
Ok, might be worthwhile just to add 'minimizing layers and reducing build times.'

Reply:
Yes, repeat build up speed... I'll make sure to clarify - thanks for the feedback.